pkg/golden/golden.go

```diff
@@ -66,7 +66,7 @@ func TestMain(m *testing.M) {
 // It can be called instead of TestMain if a package wants to do multiple
 // main function handling.
 func Setup() {
-	flag.BoolVar(&update, "golden-update", false, "update golden files")
+	flag.BoolVar(&update, "golden-update", os.Getenv("GOLDEN_UPDATE") != "", "update golden files")
```

Comment:
What do you mean? :) It's making it so that you can enable golden tests also by setting `GOLDEN_UPDATE=1`. The `-golden-update` flag is annoying because packages that don't have golden tests fail to build with it specified, `go test  ./... -golden-update` fails.

Reply:
Is your new dog keeping you up at night?

Reply:
and cat